tethys_apps/base/app_base.py

```diff
@@ -1634,6 +1635,33 @@ def persistent_store_exists(cls, name):
         ps_database_setting.persistent_store_database_exists()
         return True
 
+    def sync_all_settings(self, db_app):
+        # custom settings
+        db_app.sync_settings(self.custom_settings(), db_app.custom_settings)
+        # dataset services settings
+        db_app.sync_settings(
+            self.dataset_service_settings(), db_app.dataset_service_settings
+        )
+        # spatial dataset services settings
+        db_app.sync_settings(
+            self.spatial_dataset_service_settings(),
+            db_app.spatial_dataset_service_settings,
+        )
+        # wps settings
+        db_app.sync_settings(
+            self.web_processing_service_settings(), db_app.wps_services_settings
+        )
+        # persistent store settings
+        db_app.sync_settings(
+            self.persistent_store_settings(),
+            db_app.persistent_store_connection_settings
+            | db_app.persistent_store_database_settings,
+        )
```

Comment:
I'm unfamiliar with the `|` syntax used here. What is the intended effect?

Reply:
`db_app.persistent_store_connection_settings` and `db_app.persistent_store_database_settings` are both `QuerySets`. The `|` operator is how to join both sets.